trio/_core/tests/test_multierror.py

```diff
@@ -694,6 +694,16 @@ def test_ipython_imported_but_unused():
     check_simple_excepthook(completed)
 
 
+@slow
+@need_ipython
```

Comment:
I don't think this test actually needs IPython?

Reply:
uh...  :[